### DISCUSSION TITLE
百度网盘错误码：9019

### DISCUSSION THREAD

**Discussion by DAIII139** [AlistGo/alist#2949] [Q&A]
挂载百度网盘稳定运行了一段时间后出现下载或是播放会报 错误
errno: 9019, refer to https://pan.baidu.com/union/doc/
![image](https://user-images.githubusercontent.com/84062370/211179070-f8516fea-67f7-47d1-93fa-8506b8788b56.png)
版本是2.6.4 Alist  也已在最新版3.7.2试过也是报错9019
附上3.7.2版本报错截图
![image](https://user-images.githubusercontent.com/84062370/211179140-034c3e53-8550-4808-9968-43be52d0bc2b.png)
 
报错地址测试： https://vfx.vfx.xn--5tzm5g/vfx/blender%E8%AF%BE%E7%A8%8B

**Comment by Rayciu** [comment]
你好，请问你解决了吗？我昨天网盘充了SVIP后 也出现这样的问题

**Reply by Rayciu** [reply]
查资料发现，估计应该是下载功能被封禁了，应该是改了密码之后重新刷新令牌，还有注意不要改 ua，不然会被继续封
